src/CoffeeSwiper.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {coffee_products} from "./data/products";
+
+// Import Swiper React components
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+// Import Swiper styles
+import 'swiper/css';
+import CoffeeProductLine from "./components/CoffeeProductLine";
+import {Navigation} from "swiper";
+
+
+const CoffeeSwiper = () => {
+    return (
+        <Swiper style={{overflow:"visible"}}
+            spaceBetween={30}
+            slidesPerView={2}
+                modules={[Navigation]}
+                navigation={{
+                    prevEl: '.Coffee__swiper-button-prev-my',
+                    nextEl: '.Coffee__swiper-button-next-my',
+                }}
+        >
+            {coffee_products.map((product_line, index) =>
+                <SwiperSlide  key = {'slide number '+index}>
+                    <CoffeeProductLine line={product_line}
+                                       key = {index}></CoffeeProductLine>
+                </SwiperSlide>)}
+            <div className="swiper-button-prev-my Coffee__swiper-button-prev-my">
```

Comment:
There's a better way to import styles:
- create a `<modulename>.module.css`
- import it: 
```typescript
import styles from './modulename.module.css'
```
- use it like so:
```typescript
<div className={styles.swiperButtonPrevMy} />
```

More info here: https://create-react-app.dev/docs/adding-a-css-modules-stylesheet